lib/htmltoword/document.rb

```diff
@@ -113,16 +113,14 @@ def transform_and_replace(source, stylesheet_path, file, remove_ns = false)
     def local_images(source)
       source.css('img').each_with_index do |image,i|
         filename = image['data-filename'] ? image['data-filename'] : image['src'].split("/").last
-        ext = ext_from_filename(filename)
+        ext = File.extname(filename).delete(".").downcase
 
         @image_files << { filename: "image#{i+1}.#{ext}", url: image['src'], ext: ext }
```

Comment:
From memory using the actual file extension rather than the cleaned content type to name the local file could cause issues where the file extension is is different to the content type.

Can you double check that your PR works with image files with extensions like jpg and tif before I merge?